install.sh

```diff
@@ -229,8 +260,13 @@ get_iso()
 
 setup_style()
 {
-    if [ "$K3OS_INSTALL_FORCE_EFI" = "true" ] || [ -e /sys/firmware/efi ]; then
+    if grep -q 'k3os.install.efi_hybrid=true' /proc/cmdline ; then
```

Comment:
@dweomer What is your opinion of the hybrid GPT/MBR added to install.sh? Still not sure whether it is worth the hassle.

Reply:
@bdabelow what does it solve for?

Reply:
It allows creation of installations/disk images that will boot in BIOS/MBR as well as UEFI/GPT (virtual) machines.